docs/src/components/pricing/PricingTable.tsx

```diff
@@ -1008,16 +1008,16 @@ function RowCategory(props: BoxProps) {
           pl: 1.5,
           display: 'block',
           textTransform: 'uppercase',
-          letterSpacing: '.08rem',
+          letterSpacing: '.1rem',
           fontWeight: theme.typography.fontWeightBold,
           fontSize: theme.typography.pxToRem(11),
-          color: 'text.secondary',
+          color: (theme.vars || theme).palette.text.tertiary,
```

Comment:
I'm not sure if it was here, but the label of Data Grid and charts feels disabled in dark theme:

<img width="279" alt="Screenshot 2024-03-07 at 09 28 14" src="https://github.com/mui/material-ui/assets/37222944/aa58c01c-a67c-492c-9432-9297d410aa72">

Reply:
Good catch! Should be fixed now 👌